docs: add details for IBC Testing with Emerynet and Local Chain

rabi-siddique opened the pull request:
Addresses #7

rabi-siddique committed:
docs: add details for IBC Testing with Emerynet and Local Chain

frazarshad reviewed (commented):
the documentation looks good. but section above it (testing devnet with emerynet) is incorrect now since `docker-compose-testnet.yaml` does not contain a devnet chain anymore

rabi-siddique commented:
> the documentation looks good. but section above it (testing devnet with emerynet) is incorrect now since `docker-compose-testnet.yaml` does not contain a devnet chain anymore

Thanks for pointing this out. I've addressed this in #9 and added a description for it in #10
